spec/lib/pact_broker/api/decorators/matrix_decorator_spec.rb

```diff
@@ -89,7 +89,8 @@ module Decorators
                 environments: [
                   {
                     name: "test",
-                    displayName: "Test"
+                    displayName: "Test",
+                    applicationInstance: "test-instance"
```

Comment:
should we have an additional entry for one without applicationInstance, so we are including a test case, rather than overloading another one?

It's just to check that everything works for deployed versions where there is no applicationInstance, related to my comment above :)